test/unit/account/search_test.rb

```diff
@@ -126,6 +125,8 @@ class Account::SearchTest < ActiveSupport::TestCase
       ThinkingSphinx::Test.config
       ThinkingSphinx::Test.index verbose: true
```

Comment:
We can remove the verbose

Reply:
The verbose generates something like this:

```
indexing index 'account_core'...
collected 2 docs, 0.0 MB
sorted 0.0 Mhits, 100.0% done
total 2 docs, 1392 bytes
total 0.143 sec, 9716 bytes/sec, 13.96 docs/sec
indexing index 'account_delta'...
collected 2 docs, 0.0 MB
sorted 0.0 Mhits, 100.0% done
total 2 docs, 1392 bytes
total 0.176 sec, 7881 bytes/sec, 11.32 docs/sec
indexing index 'cms_page_core'...
collected 0 docs, 0.0 MB
total 0 docs, 0 bytes
total 0.202 sec, 0 bytes/sec, 0.00 docs/sec
indexing index 'topic_core'...
collected 0 docs, 0.0 MB
total 0 docs, 0 bytes
total 0.232 sec, 0 bytes/sec, 0.00 docs/sec
indexing index 'topic_delta'...
collected 0 docs, 0.0 MB
total 0 docs, 0 bytes
total 0.202 sec, 0 bytes/sec, 0.00 docs/sec
skipping non-plain index 'account'...
skipping non-plain index 'cms_page'...
skipping non-plain index 'topic'...
total 19 reads, 0.000 sec, 0.1 kb/call avg, 0.0 msec/call avg
total 45 writes, 0.001 sec, 0.1 kb/call avg, 0.0 msec/call avg
rotating indices: successfully sent SIGHUP to searchd (pid=87555).
```

Reply:
Removed, though keeping the option to use it with the new `Indexer` helper.